src/context/auth-context/AuthContainer.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useContext } from 'react';
+import { AuthContext, AuthProvider } from './AuthContext';
+
+
+const AuthContextContainer = ({ children }) => {
+  const [isAuthenticated, setIsAuthenticated] = useState(
+    JSON.parse(localStorage.getItem('isAuthenticated')) ?? false
+  );
+  const setLogin = () => {
```

Comment:
We should pass token as a parameter to this function from login page.

Reply:
we currently don't have a BE to generate an auth token. So I guess there is no need to store the token if that's what you mean. Otherwise, can you kindly explain?